packages/fxa-auth-db-mysql/scripts/readSecurityEvents.js

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env node
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * This script takes the uid of a user and will print a security event object of that user
+ * from various tables like securityEvents, sessionTokens and devices tables from fxa database
+ */
+
+// Usage: node readSecurityEvents.js uid
+
+const log = {
+  trace: console.log,
+  error: console.error,
+  stat: console.log,
+  info: console.log
+} //eslint-disable-line no-console
+const DB = require('../lib/db/mysql')(log, require('../db-server').errors)
+const config = require('../config')
+
+const securityEventsObject = {}
+
+// This input of user's data can be modified according to the usecase
+const user = {
+  // email: 'farhan@gmail.com',
+  // uid: '2A4A31EBAC9E46D9AD483B92789EC934',
+  uid: process.argv[2],
+  ipAddr: '127.0.0.1'
+}
+
+function readEvents() {
+  DB.connect(config).then(async db => {
+    const devices = await readDevices(db)
+    securityEventsObject.devices = devices
+
+    const sessions = await readSessionTokens(db)
+    securityEventsObject.sessions = sessions
+
+    const events = await readSecurityEvents(db)
+    securityEventsObject.securityEvents = events
+
+    console.log(securityEventsObject) //eslint-disable-line no-console
+
+    db.close()
+  })
+}
+
+function readDevices(db) {
+  let { uid } = user
+  uid = Buffer.from(uid, 'hex')
+
+  return db
+    .accountDevices(uid)
+    .then(devices => {
```

Comment:
This `.then` doesn't really do anything, tokens will be returned by the promise already.